CMakeLists.txt

```diff
@@ -63,11 +63,16 @@ target_include_directories(maeparser PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
     $<INSTALL_INTERFACE:include>)
 
+if(MSVC)
```

Comment:
Why is this MSVC block here? If it's not present, are the CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_BINDIR unreliable?

Reply:
CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_BINDIR are set by [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html).  Presumably, this isn't set in MSVC since this PR was failing CI in MSVC without those variables being set.